### ISSUE TITLE
Missing Translations Files for Linux Build

### ISSUE THREAD

**Issue by banduccm** [mozilla-mobile/mozilla-vpn-client#872]
I'm attempting to build mozillavpn on Fedora 33, and while I've worked around a few things, I'm stuck at what appears to be a missing file:
```
[user@localhost mozilla-vpn-client]$ sudo make install
[sudo] password for user: 
cd src/ && ( test -e Makefile || /usr/bin/qmake-qt5 -o Makefile /home/user/repos/mozilla-vpn-client/src/src.pro CONFIG+=production ) && make -f Makefile install
make[1]: Entering directory '/home/user/repos/mozilla-vpn-client/src'
lupdate /home/user/repos/mozilla-vpn-client/src -no-obsolete -ts /home/user/repos/mozilla-vpn-client/src/../translations/mozillavpn_en.ts
make[1]: lupdate: No such file or directory
make[1]: *** [Makefile:971: ts] Error 127
make[1]: Leaving directory '/home/user/repos/mozilla-vpn-client/src'
make: *** [Makefile:60: sub-src-install_subtargets] Error 2
```

I followed the build instructions verbatim:
```
qmake CONFIG+=production
make -j8 # replace 8 with the number of cores. Or use: make -j$(nproc)
sudo make install
```

Is `/translations/mozillavpn_en.ts` missing, or is there a step missing from the build process to generate it?

**Comment by birdsarah** [comment]
There may well be a step missing from the instructions. Does running `python3 scripts/importLanguages.py` or `./scripts/importLanguages.py` help?

**Comment by banduccm** [comment]
I think I'm getting closer! I ran that script and I noticed that `lupdate` on Fedora is `lupdate-qt5`, so I may have some more tweaking to do. I was able to generate the files in /translate/, so I'm thinking that the build instructions are correct and I'm running into some stumbling blocks because I'm on a different distro. I'll keep playing with it and report back when I have a real answer.

Thank you for the quick response!
